olive/evaluator/olive_evaluator.py

```diff
@@ -1017,14 +1015,13 @@ def _inference(
         preds = []
         targets = []
         logits = []
-        for input_data, labels in dataloader:
-            result = session.infer_new_request({0: input_data})
+        for input_data, label in dataloader:
+            session.infer({0: input_data})
+            result = session.get_output_tensor(0).data
             outputs = post_func(result) if post_func else result
-            if not isinstance(labels, list):
-                labels = [labels]  # noqa: PLW2901
-            preds.extend(outputs)
-            targets.extend(labels)
-            logits.extend(result)
+            preds.append(outputs)
```

Comment:
is there a reason this is `append` and not `extend`. Is the data not in batches for openvino? Just one sample per iteration of the loop?

Reply:
Ah good catch! I made this change to test torchmetrics accuracy api feature but forgot to change it back. Will update this.